website/templates/emails/prereg_reminder.html.mako

```diff
@@ -0,0 +1,16 @@
+## -*- coding: utf-8 -*-
+<%inherit file="notify_base.mako"/>
+<%def name="content()">
+    <div style="margin: 40px;">
+        <br>
+        Dear ${fullname},
+        <br><br>
+        You have an unsubmitted preregistration on the Open Science Framework that could be eligible for a $1,000 prize as part of the <a href="https://cos.io/prereg">Prereg Challenge</a>. If you would like this to be considered for a prize, please complete <a href="${prereg_url}">your preregistration</a> and submit it for review, available on the last page of the preregistration form. This review process is required for a prize and simply checks to make sure the preregistration includes a complete analysis plan.
+        If you have questions about a particular field in the preregistration, you may review the <a href="https://cos.io/prereg">FAQ on the website</a>, <a href="mailto:prereg@cos.io">email us with a question</a>, or use our <a href="https://cos.io/stats_consulting">free statistical consulting services</a>.
+        Thank you for using the OSF!
+        <br><br>
+        Sincerely,
+        <br>
+        The team at the Center for Open Science
```

Comment:
Does this have the address somewhere in it? It might be in the full email, but if so, the footer was cut off on the screen shot.

Reply:
Ya, sorry. The address is in fact there.